content/content-strategy/define-user-content-needs/cards.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: cards/cards
+cards:
+  - headline: Understand users and their needs
+    text: Knowing your users and their needs is the starting point for any product or service.
+    link: https://guides.service.gov.au/user-research/identifying-users-needs/
+  - headline: Research and learn from users
+    text: How to plan for research, adapt your methods, recruit and interview, and analyse what you learn.
+    link: https://guides.service.gov.au/user-research/
```

Comment:
Use relative path, `../user-research/` instead of absolute

Reply:
Steve fixed